### DISCUSSION TITLE
Help How i run software in cli mode

### DISCUSSION THREAD

**Discussion by DiamondMagpie** [LETHAL-FORENSICS/MemProcFS-Analyzer#22] [Q&A]
Hello, how can I run the MemProcFS-Analyzer software in cli mode? 
I want to start the software by powershell command without all the gui but only have as output the files analysis

**Comment by evild3ad** [comment]
This is currently not supported...and not planned yet.

When you are familiar with PowerShell you can remove the user interface code block and edit the 1 or 2 variables always manually before launching via command line and/or add optional arguments.

MemProcFS-Analyzer v0.9 will be released this week...with an updated user interface with more features:

![23-05-2023_15-05-26](https://github.com/evild3ad/MemProcFS-Analyzer/assets/8110531/535e291d-e3fc-4cbb-947a-89b8959a582b)

**Comment by DiamondMagpie** [comment]
Thanks ! <3
